shared-data/protocol/schemas/7-draft.json

```
@@ -518,6 +518,23 @@
             }
           },
 
+          {
+            "description": "[This command is for internal use only and should NOT be used in protocols.] Initiate automatic calibration of a pipette.",
+            "type": "object",
+            "required": ["commandType", "params"],
+            "properties": {
+              "key": { "type": "string" },
+              "commandType": { "enum": ["calibration/calibratePipette"] },
+              "params": {
+                "required": ["mount"],
+                "labwareId": {
```

```suggestion
                "mount": {
```
Should this be mount?

---

yep thanks! leftover from a copy/paste